doc/user_guide/user_guide.md

```diff
@@ -1,68 +1,133 @@
+
+**Important note for Windows:** Make sure to download and install file `EXASOL_JDBC-<version>.msi`. This will install the JDBC driver to `C:\Program Files\Exasol\EXASolution-7.1\JDBC\exajdbc.jar`. Only the JDBC driver for Windows supports Kerberos under Windows.
+
+Copy the JDBC driver `exajdbc.jar` to the Tableau installation directory:
+
+* Tableau Desktop (see [Tableau Desktop documentation](https://help.tableau.com/current/pro/desktop/en-us/examples_otherdatabases_jdbc.htm#specify-the-right-jdbc-driver) for details):
+  * `C:\Program Files\Tableau\Drivers` (Windows)
+  * `~/Library/Tableau/Drivers` (macOS)
+* Tableau Server:
   * Windows: `C:\Program Files\Tableau\Drivers`
-  * Mac: `~/Library/Tableau/Drivers`
   * Linux: `/opt/tableau/tableau_driver/jdbc`
 
-**Important note for Windows:** Make sure to download and install file `EXASOL_JDBC-<version>.msi`. The JDBC driver will be installed to `C:\Program Files\Exasol\EXASolution-7.1\JDBC\exajdbc.jar`. Only the JDBC driver for Windows supports Kerberos under Windows.
+#### Install the ODBC Driver
 
-### With Tableau Desktop
+The connector for ODBC requres the Exasol ODBC driver to be installed for Tableau Desktop and Server.
 
-* Clone the Tableau-Connector repository. You can execute the following command from [Git Bash](https://gitforwindows.org/) terminal:
+Download and install the latest JDBC driver for your operating system from the [download page](https://www.exasol.com/portal/display/DOWNLOAD/).
 
-    ```shell
-    git clone https://github.com/exasol/tableau-connector.git
-    ```
+* Windows and macOS: install the driver by executing the installer.
+* Linux:
+  1. Unpack the ODBC driver to `/opt/exasol/odbc`
+  2. Create or edit file `/etc/odbcinst.ini` and add the following entry:
 
-* Open Windows cmd and start Tableau Desktop providing the path to the connector directory and also disabling the signature check.
+      ```ini
+      [EXASolution Driver]
+      Driver=/opt/exasol/odbc/lib/linux/x86_64/libexaodbc-uo2214lv2.so
+      ```
 
-    ```shell
-    "<path to tableau.exe file>" -DConnectPluginsPath=<path to connector>
-    ```
+### Install the Connectors With Tableau Desktop
 
-    Example:
+1. Copy the connector `.taco` files to
+  * `C:\Users\[Windows User]\Documents\My Tableau Repository\Connectors` (Windows)
+  * `/Users/[user]/Documents/My Tableau Repository/Connectors` (macOS)
+2. As the connectors are not signed, you need to start Tableau Desktop with command line argument `-DDisableVerifyConnectorPluginSignature`, e.g. by creating a `.bat` file with the following content:
 
-    ```shell
-    "C:\Program Files\Tableau\Tableau <version>\bin\tableau.exe" -DConnectPluginsPath=C:\Users\user\git\tableau-connector\src
+    ```bat
+    "C:\Program Files\Tableau\Tableau 2021.3\bin\tableau.exe" -DDisableVerifyConnectorPluginSignature
```

Comment:
Maybe the version placeholder is better? You can forget to change this when the new Tableau version is released